test/util/aleTests/ALEyvyCRVHelperForkTest.t.sol

```diff
@@ -337,9 +288,10 @@ contract ALEHelperForkTest is Test {
 
         // Balance in escrow is equal to the collateral deposited + the extra collateral swapped from the leverage
         assertApproxEqAbs(
```

Comment:
Consider adding asserts that check balance of escrow and that `maxBorrowAmount` hasn't been exceeded.